ssm_jax/hmm/demos/low-rank_gaussian_hmm.ipynb

```diff
@@ -0,0 +1,275 @@
+{
```

Comment:
<p>Maybe it's just this notebook viewer, but can you run the cells so that we can see the outputs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/ssm-jax/pull/157/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Done ! These should now be in the notebook.